src/initial-submission/components/EditorsForm.tsx

```diff
@@ -56,6 +57,11 @@ const EditorsForm = ({ initialValues, ButtonComponent }: StepProps): JSX.Element
                 [['name', 'email']],
             ),
         ),
+        opposedReviewingEditors: yup.array().max(2, t('opposed-reviewering-editors-max')),
+        opposedReviewersReason: yup.string().when('opposedReviewingEditors', {
+            is: editors => !!editors.length,
+            then: yup.string().required(t('editors.validation.opposed-reviewering-editor-reason-required')),
```

Comment:
Can we get a unit test to check this validation message is shown if its left blank after selecting an editor?

Reply:
see https://github.com/libero/reviewer-client/blob/511e475ba39ec709e43b37b8bc4bd04fa419163e/src/initial-submission/components/EditorsForm.test.tsx#L451 for how we've done that for other inputs on page

Reply:
Makes sense. Was going to do that as part of the end to end test.

Reply:
The test bellow that is probably a better example (as that ones a test to ensure no validation is shown). Using the initialValues to load in with opposed Reviewers but have no reason value will be your best bet